crates/loans/src/lend_token.rs

```diff
@@ -141,7 +141,10 @@ impl<T: Config> Pallet<T> {
         let reducible_underlying_amount = reducible_supply_amount.convert_to(underlying_id)?.amount();
 
         let exchange_rate = Self::exchange_rate(underlying_id);
-        let amount = Self::calc_collateral_amount(reducible_underlying_amount, exchange_rate)?;
+        let amount = FixedU128::from_inner(reducible_underlying_amount)
```

Comment:
Because it doesn't exist anymore - I had removed it since no code seemed to depend on it anymore after my refactoring. But then it turned out this test code still used it.. I didn't want to re-add the function since no non-testing code should use it

Reply:
You could also move it to `lend_token.rs` where the other test utilities are